bitnami/redis/templates/master/application.yaml

```
@@ -26,7 +26,7 @@ spec:
   {{- else }}
   updateStrategy: {{- toYaml .Values.master.updateStrategy | nindent 4 }}
   {{- end }}
-  {{- if .Values.master.minReadySeconds }}
+  {{- if .Values.master.minReadySeconds (if semverCompare ">= 1.25" (include "common.capabilities.kubeVersion" .)) }}
```

Do the nested `if` work? Shouldn't it be an `and` statement?
```suggestion
  {{- if and .Values.master.minReadySeconds (semverCompare ">= 1.25" (include "common.capabilities.kubeVersion" .)) }}
```